lib/generators/rollbar/templates/initializer.rb

```diff
@@ -1,4 +1,3 @@
-require 'rollbar/rails'
```

Comment:
The file has not been deleted, so the require itself doesn't generate any error. That file just configured the notifier for rails, `framework`, `logger`, etc...But that is actually done in the railtie, what is enough.